src/arch/x86/kernel/boot_sys.c

```diff
@@ -452,9 +475,7 @@ static BOOT_CODE bool_t try_boot_sys(void)
     assert(mods_end_paddr > boot_state.ki_p_reg.end);
 
     printf("ELF-loading userland images from boot modules:\n");
-    load_paddr = mods_end_paddr;
-
-    load_paddr = load_boot_module(boot_state.boot_module_start, load_paddr);
+    load_paddr = load_boot_module(boot_state.boot_module_start, &mods_end_paddr);
```

Comment:
`mods_end_paddr` is a local variable, you are not allowed to take its address in seL4. See also <https://docs.sel4.systems/processes/style-guide.html#verification-requirements>